ci/infra/vmware/worker-instance.tf

```diff
@@ -67,6 +67,7 @@ resource "vsphere_virtual_machine" "worker" {
   scsi_type        = data.vsphere_virtual_machine.template.scsi_type
   resource_pool_id = data.vsphere_resource_pool.pool.id
   datastore_id     = data.vsphere_datastore.datastore.id
+  datastore_cluster_id = data.vsphere_datastore_cluster.datastore.id
```

Comment:
cant use both. If `datastore_id` and `datastore_cluster_id` is set at the same time, deployment fails:
`Error: "datastore_cluster_id": conflicts with datastore_id`